src/js/stores/AppFormStore.js

```diff
@@ -3,7 +3,8 @@ var lazy = require("lazy.js");
 var Util = require("../helpers/Util");
 
 var AppDispatcher = require("../AppDispatcher");
-var AppFormTransforms = require("./AppFormTransforms");
+var AppFormFieldToModelTransforms = require("./AppFormFieldToModelTransforms");
+var AppFormModelToFieldTransforms = require("./AppFormModelToFieldTransforms");
```

Comment:
Seriously though, what if we merged these into an `AppFormTransforms` and exported 
```
{
   FieldsToModel: { // ... }
   ModelToFields: { //... }
}
```
?